Library/Homebrew/download_strategy.rb

```diff
@@ -714,7 +714,10 @@ def stage
 class LocalBottleDownloadStrategy < AbstractFileDownloadStrategy
   def initialize(path) # rubocop:disable Lint/MissingSuper
     @cached_location = path
-    extend Pourable
```

Comment:
Is there a reason to remove the `Pouring` output?

Reply:
No, this only changes the `stage` output, so can be reverted.